modules/fts/pages/fts-rebalance-failover.adoc

```diff
@@ -62,3 +62,21 @@ The index partitions residing on the recovered node would be reused during a rec
 3. Perform the software update or hardware maintenance operations.
 4. Perform recover rebalance operation.
 5. Cluster is back to normal/pre-failover safe state.
+
+=== Why rebalance button remains active even after a successful rebalance
+
+Even after a successful Search rebalance operation, if the rebalance button remains `active` then it hints that the Search cluster is not having enough nodes
+to support the number of replicas configured for the Search index. The Search service considers the cluster is still imbalanced as it isn't having enough
+hardware resources to support the set configuration of replica count for the index partitions.
```

Comment:
I'd recommend a re-phrase here ..

..if the rebalance button remains `active` then it is indicative of the cluster not containing enough nodes that host the _Search_ service to support the configured number of replicas for the index.

Also, I'd drop the 2nd sentence which feel repetitive.

Reply:
👍